libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts

```diff
@@ -56,6 +49,11 @@ export class SdsFiltersComponent implements OnInit {
    */
   @Input() public options: FormlyFormOptions = {};
 
+  /**
+   *   Display Reset All button that returns form fields to default states -- default false.
+   */
+  @Input() public resetAll: boolean = false;
```

Comment:
Per Christy, this should be always enabled. So it should always be set to true by default, and we may end up removing this option completely.